examples/generate_image_dataset/generate_images.py

```diff
@@ -107,7 +107,7 @@ def single_step(self, action=None, get_observation=False):
             # create dir for storing images
             if not args.benchmark:
                 for render_pass in config.SIMULATION_CONTROLLER.CAMERA_AGENT.CAMERA.RENDER_PASSES:
-                    render_pass_dir = os.path.realpath(os.path.join(args.images_dir, render_pass))
+                    render_pass_dir = os.path.realpath(os.path.join(args.images_dir, pose["scene_id"], render_pass))
```

Comment:
I don't think this added level of hierarchy is necessary. We're using a global index for the filename that corresponds directly to the row in poses.csv, so I think it's slightly cleaner to have all the images in a single folder.